### PR TITLE
Animate expandable table rows

### PR DESCRIPTION
**Pull request by FlyersPh9**
- When a table row expands / collapses it animates.
- Fixed a bottom border of a selected expanded row.
- All row borders transition.

### CODE REVIEW

File: src/table/table.scss [hunk 1]
```diff
@@ -161,6 +161,20 @@
...
     }
 
+    &.iui-expanded-content {
+      transform: scaleY(0);
+      transform-origin: top;
+      transition: transform $iui-speed-fast ease-in-out;
+      overflow: hidden;
+      opacity: 0;
+      height: 0;
+      margin-top: -2px;
```
Comment (FlyersPh9) [inline src/table/table.scss:171]: Unsure why this is needed.

File: src/table/table.scss [hunk 2]
```diff
@@ -161,6 +161,20 @@
       visibility: visible;
     }
 
+    &.iui-expanded-content {
+      transform: scaleY(0);
+      transform-origin: top;
+      transition: transform $iui-speed-fast ease-in-out;
+      overflow: hidden;
+      opacity: 0;
```
Comment (mayank99) [inline src/table/table.scss:169]: This approach will not work.

This is what the DOM currently looks like:
table

I'm making some changes in React that should give you classes for animating.

Comment (FlyersPh9) [reply src/table/table.scss]: We collaborated and are now using the correct classes.

**Review by mayank99 — APPROVED** [review]
Works perfectly with https://github.com/iTwin/iTwinUI-react/pull/169 🎉

File: src/table/table.scss [hunk 3]
```diff
@@ -180,6 +186,24 @@
       }
     }
 
+    &.iui-expanded-content {
```
Comment (bentleyvk) [inline src/table/table.scss:189]: Can we have this in HTML as an example?

Comment (mayank99) [reply src/table/table.scss:189]: These classes come from react-transition-group, so I think not

**Pull request merged**